src/aura/AuraAvatarTwoToken.sol

```diff
@@ -304,6 +334,20 @@ contract AuraAvatarTwoToken is
         emit MinOutBpsBalToBptMinUpdated(_minOutBpsBalToBptMin, oldMinOutBpsBalToBptMin);
     }
 
+    function setTwapPeriod(uint256 _twapPeriod) external onlyOwner {
```

Comment:
After further looking, I noticed that this can't be 0 and we should probably enforce that for safety as getting the average price from the pool would [revert](https://etherscan.io/address/0x5c6Ee304399DBdB9C8Ef030aB642B10820DB8F56#code#F1#L798) and it will break the processing function.